docs/getting-started.md

```diff
@@ -54,9 +75,11 @@ In bovenstaand figuur is ook te zien hoe de `expand` Query Parameter eruit moet
 Het is mogelijk om alleen specifieke kenmerken van een ingeschreven persoon te bevragen met behulp van de `fields` Query Parameter. In onderstaand figuur is een bevraging te zien met de `fields` Query Parameter gezet op `burgerservicenummer,naam,geboorte.datum`.
 ![fields parameter](./img/9-using-fields-parameter.jpg)
 
-Onderstaand tabel bevat de bsn van test personen voor specifieke situaties waarmee werking van de 'Bevraging Ingeschreven Persoon' Web API kan worden getest.
+### Test Personen
+
+Onderstaande tabel bevat de burgerservicenummers van testpersonen voor specifieke situaties waarmee werking van de 'Bevraging Ingeschreven Persoon' Web API kan worden getest.

```

Comment:
burgerservicenummers = burgerservicenummer?

Reply:
"de burgerservicenummers van testpersonen". Kan dat niet?

Je kan toch ook zeggen "de fietsen van de scholieren" of "hier liggen de laptops van de developers" of "in de tabel staan de namen van de deelnemers"

Reply:
ik lees/las: een persoon heeft meerdere burgerservicenummers, maar jouw andere voorbeelden zijn zoals ik het ook zou opschrijven. Laat het maar op burgerservicenummers.